vars/daosPackagesVersion.groovy

```diff
@@ -60,7 +60,7 @@ String call(String distro, String next_version) {
         return version + dist
     }
 
-    if (target_branch.matches('^[a-z]+-testing')) {
+    if (target_branch.matches('^[-0-9A-Za-z]+-testing')) {
```

Comment:
This (and all of the others below) could be DRYed out with:

```suggestion
    if (target_branch.matches(testBranchRE()) {
```

if you were to create a new `vars/testBranchRE.groovy` variable as such:

```groovy
// vars/testBranchRE.groovy

  /**
   * testBranchRE step method
   */

def call() {
    return '^[-0-9A-Za-z]+-testing'
}
```

Seems like overkill for just a global/constant variable and there is probably a better way to do it, but it's just the way I know of off-hand.

At least this way we only have to modify the RE in two places instead of the numerous places it is used.